docs/topics/developer-tools.rst

```diff
@@ -254,15 +254,40 @@ page-number greater than 10), we increment the ``page`` attribute
 and ``yield`` a new request, inserting the incremented page-number 
 into our ``url``. 
 
-You can see that with a few inspections in the `Network`-tool we 
+In more complex websites, it could be difficult to easily reproduce the
```

Comment:
This is what I meant in my comment about the `:ref:` link on the dynamic content documentation:

```suggestion
.. _requests-from-curl::

In more complex websites, it could be difficult to easily reproduce the
```